ios/PalmApp/PalmApp-Testnet-Info.plist

```diff
@@ -32,7 +32,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>28</string>
+	<string>31</string>
```

Comment:
why suddenly bumped to 31?

Reply:
build was uploaded to the appstore several times, so  bundleversion was increased to upload it again.

Reply:
can you please use `fastlane ios/android bump_build_version` from next time? otherwise, likely to break versioning.